components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -57,14 +57,21 @@ def _configure(self, parser):
         sub = parser.sub()
 
         parser.add(sub, self.help, "Extended help")
-        parser.add(sub, self.start, "Primary start for the OMERO.web server")
+        start = parser.add(
+            sub, self.start, "Primary start for the OMERO.web server")
         parser.add(sub, self.stop, "Stop the OMERO.web server")
-        parser.add(sub, self.restart, "Restart the OMERO.web server")
+        restart = parser.add(
+            sub, self.restart, "Restart the OMERO.web server")
         parser.add(sub, self.status, "Status for the OMERO.web server")
 
         iis = parser.add(sub, self.iis, "IIS (un-)install of OMERO.web ")
         iis.add_argument("--remove", action="store_true", default=False)
 
+        for x in (start, restart, iis):
+            x.add_argument(
+                "--skip-clear-sessions", action="store_true",
```

Comment:
Also works. That being said with the improvements made following the suggestions of @aleksandra-tarkowska/@manics (i.e. clearing sessions using `popen` rather than `call`), maybe we do not need this option at all?